README.md

```diff
@@ -170,105 +201,81 @@ These instructions have been written with a Debian 12 machine in mind, but they
-ln -s soundemittersystem_srv.so soundemittersystem.so;
-ln -s shaderapiempty_srv.so shaderapiempty.so;
-ln -s scenefilecache_srv.so scenefilecache.so;
-ln -s replay_srv.so replay.so;
-ln -s materialsystem_srv.so materialsystem.so;
-```
-6. Linux-only: Before running `srcds_linux`, some few environment variables need to setup:
-    * `SteamEnv=1`
-    * `LD_LIBRARY_PATH=$(<STEAM-RUNTIME-DIR>/run.sh printenv LD_LIBRARY_PATH):/home/YOUR_USER/.steam/steam/steamapps/Source SDK Base 2013 Dedicated Server/bin`
-        * Where `<STEAM-RUNTIME-DIR>` can be found from: `$ find "$HOME" -type d -name 'steam-runtime' 2> /dev/null`
-7. Run: `<srcds.exe|srcds_linux> +sv_lan 0 -insecure -game neo +map <some map> +maxplayers 24 -autoupdate -console`
-    * Double check on the log that VAC is disabled before continuing
-7. In-game on Windows it'll showup in the server list, on Linux it probably won't and you'll have to use `connect` command directly (EX: `connect 192.168.1.###` for LAN server)
+* **Windows-only:** Allow all Inbound and Outbound TCP and UDP requests for the given ports via Windows Firewall. [See how](https://learn.microsoft.com/en-us/windows/security/operating-system-security/network-security/windows-firewall/configure#create-an-inbound-port-rule)
+
+##### Steam Networking
+
+Since the TF2-SDK update, NT;RE gained the ability to use "Steam Networking" which can be useful in situations where:
+
+* You cannot setup port forwarding or/and altering firewall settings
+* You don't want to expose the server's IP address and device directly
+
+Just add `+sv_use_steam_networking 1` to the `srcds` command line arguments to use Steam Networking for your dedicated server.
+
+#### Running the server
+You will need to be in the `<YOUR_LOCATION>/ntrebuild` directory and run `srcds_win64`/`srcds_run_64`/`srcds_linux64` with whatever arguments. You can adapt the following command to your own liking:
+
+* Windows:
+    ```
+    srcds_win64.exe -game neo -neopath "<YOUR_LOCATION>\ognt\NeotokyoSource" +sv_lan 0 -insecure +ip <YOUR_IP> -maxplayers <1-32> +map <MAP_NAME>
+    ```
+* Linux with `srcds_run_64` script:
+    ```
+    ./srcds_run_64 -game neo +sv_lan 0 -insecure -console +ip <YOUR_IP> -maxplayers <1-32> +map <MAP_NAME>
+    ```
+* Linux with `srcds_linux64` directly:
+    1. Before running `srcds_linux64`, some few environment variables need to setup:
+        * `SteamEnv=1`
+        * `LD_LIBRARY_PATH="<YOUR_LOCATION>/ntrebuild/bin/linux64"` - So it can find the server libraries
+    2. Run: `srcds_linux64 -game neo +sv_lan 0 -insecure -console +map <some map> +maxplayers 24`
+        * In-game it probably won't showup in the server list and you'll have to use `connect` command directly (EX: `connect 192.168.1.###` for LAN server)
+
+If using `-insecure` and intend to playtest without VAC for debugging purposes, double check on the log that VAC is disabled before continuing.
 
 ### SourceMod server modification plugins
```

Comment:
Sourcemod is broken until metamod patches come in for the 64bit update to the sdk.


https://github.com/alliedmodders/metamod-source/commit/dc41559c7905072feec38b6d45ea14c05da3b855